iceberg/iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandlerTestUtils.java

```diff
@@ -126,6 +126,10 @@ static TestTables testTables(TestHiveShell shell, TestTables.TestTableType testT
     return testTableType.instance(shell.metastore().hiveConf(), temp, catalogName);
   }
 
+  static void init(TestHiveShell shell, TestTables testTables, TemporaryFolder temp) {
+    init(shell, testTables, temp, "tez");
```

Comment:
Should an enum be used here instead of hardcoded "tez" value?

Reply:
LGTM
Added a few comments and questions

Reply:
I don't think we have enum for that + I reused the same approach as in the test.